### PR TITLE
add(platform): AE updates for networking features

### PR DESCRIPTION
**Pull request by wojcik-dorota**
https://aiven.atlassian.net/browse/DOC-769

1. Pro Platform tags added to the document titles
2. Pro Platform requirement added to the "Prerequisites" sections

Affected documents:

1. AWS privatelink
2. AWS transit GW
3. Azure Privatelink
4. GCP PSC
5. Enhanced Compliance Envirs

## Checklist

- [x] The first paragraph of the page is on one line.
- [x] The other lines have a line break at 90 characters.
- [x] I checked the output.
- [x] I applied the [style guide](../styleguide.md).
- [x] My links start with `/docs/`.

### CODE REVIEW

File: docs/platform/concepts/enhanced-compliance-env.md [hunk 1]
```diff
@@ -91,17 +93,17 @@ Although not exhaustive, Aiven is capable of supporting both the Health
 Insurance Portability and Accountability Act (HIPAA) and the Payment
 Card Industry Data Security Standard (PCI DSS) compliances.
 
-If you require compliance beyond these please contact our sales
-department so we can better understand your specific needs.
+If you require compliance beyond these, contact the [sales team](mailto:sales@aiven.io)
```
Comment (caydinavn) [inline docs/platform/concepts/enhanced-compliance-env.md:96]:
```suggestion
If you have other compliance requirements, contact the [sales team](mailto:sales@aiven.io)
```

**Review by caydinavn — APPROVED** [review]

File: docs/platform/howto/attach-vpc-aws-tgw.md [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
-attached - directly or indirectly - to the Transit Gateway.
+[AWS Transit Gateway (TGW)](https://aws.amazon.com/transit-gateway/) enables transitive routing from on-premises networks through VPN and from other VPC.
 
-## Set up a project VPC
+## How it works
 
-Create a VPC on the Aiven platform in the same region as your Transit
-Gateway.
+By creating a Transit Gateway VPC attachment, services in an Aiven Project VPC can route
+traffic to all other networks attached - directly or indirectly - to the Transit Gateway.
```
Comment (ArthurFlag) [inline docs/platform/howto/attach-vpc-aws-tgw.md:11]: Could we avoid using dashes? If they can be replaced by commas, let's do so. These dashes are not meant for this kind of use, but only for compound nouns.

**Review by ArthurFlag — CHANGES_REQUESTED** [review]

File: docs/platform/howto/use-azure-privatelink.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Use Azure Private Link with Aiven services
+pro: ture
```
Comment (ArthurFlag) [inline docs/platform/howto/use-azure-privatelink.md:3]:
```suggestion
pro: true
```

**Review by ArthurFlag — CHANGES_REQUESTED** [review]
minor typo and we're gtg

**Pull request closed without merging**